File: CursoCsharpUdemy/exer/outros/Exer14.cs
/*
*   14) Escreva um programa que leia dois números e apresente a diferença do
*   maior para o menor.
*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer14
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Digite dois números:");
            int num1 = int.Parse(Console.ReadLine());
            int num2 = int.Parse(Console.ReadLine());

            int diff = Math.Abs(num1 - num2);

            if (num1 > num2)
            {
                Console.WriteLine("A diferença entre o maior e o menor é: " + diff);
            }
            else if (num2 > num1)
            {
                Console.WriteLine("A diferença entre o maior e o menor é: " + diff);
            }
            else
            {
                Console.WriteLine("Os números são iguais, a diferença é 0.");
            }
        }
    }
}

File: Curso de Java/Exer/src/Outros/lista64Exer/Exer14.java

/*******************************************************************************
* 14) Escreva um programa que leia dois números e apresente a diferença do
* maior para o menor.
*******************************************************************************/

import java.util.Scanner;

public class Exer14 {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		System.out.println("Primeiro número: ");
		int number1 = sc.nextInt();

		System.out.println("Segundo número: ");
		int number2 = sc.nextInt();

		double diferenca = Math.abs(number1 - number2);
		if (number1 > number2) {
			System.out.printf("A diferença entre %.2f e %.2f é %.2f.%n", number1, number2, diferenca);
		} else if (number2 > number1) {
			System.out.printf("A diferença entre %.2f e %.2f é %.2f.%n", number2, number1, diferenca);
		} else {
			System.out.println("Os números são iguais, não há diferença.");
		}

		sc.close();
	}

}
